doc/community/contributing.md

```diff
@@ -0,0 +1,111 @@
+
+You could differentiate between two elements:
+
+1. Non-automated doc elements. They simply make the text appealing. Example in `installation.md` in our doc folder. To write these requires some knowledge on writing the text which is quite easy to learn having this [cheat sheet](https://www.markdownguide.org/cheat-sheet/) in close reach.
+2. Automated doc elements using `mkdocstrings`. What they do is basically to link the code script with the doc texts, for instance, compare the python script [add_electricity.py](https://github.com/pypsa-meets-earth/pypsa-earth/blob/main/scripts/add_electricity.py) with the `api-reference/index.md` documentation. To write these kind of automation, get inspiration from our `api-reference` documentation. Further, to help understanding how things work the official [mkdocstrings documentation](https://mkdocstrings.github.io/) might help too.
+
+We found three important files/file groups for the documentation:
+
+1. `mkdocs.yml`. This is the configuration file for MkDocs and defines the navigation structure.
+2. The `.py` script with the actual code documentation (docstrings).
+
+The images for documentation should be placed into [documentation](https://github.com/pypsa-meets-earth/documentation) repository to the folder "doc/img". The content of the folder "documentation/doc/img/" is copied into "pypsa-earth/doc/img/" during building PyPSA-Earth documentation.
+
+Please, if you have problems with the documentation create an issue and let us know
+
+### How to build it locally
+
+To create the documentation locally, you need [mkdocs](https://www.mkdocs.org/). It can be installed using specifications
+from `doc/requirements.txt`. First, we recommend creating a fresh conda environment and activate it:
+
+```bash
+.../pypsa-earth-status % conda create --name pypsa-earth-docs python
+```
+
+```bash
+.../pypsa-earth-status % conda activate pypsa-earth-docs
+```
+
+Next, install the packages specified in `doc/requirements.txt` using `pip`:
+
+```bash
+.../pypsa-earth-status % pip install -r doc/requirements.txt
+```
+
+Once installation is completed, the following commands allow you to create the documentation locally:
+
+```bash
+.../pypsa-earth-status (pypsa-earth-docs) % cd doc
+```
+
+```bash
+.../pypsa-earth-status/doc (pypsa-earth-docs) % mkdocs serve
+```
+
+This will start a local server, usually at `http://127.0.0.1:8000/`.
+VScode provides a so called Liveserver extension such that the html file can be opened locally on your computer.
+
+The documentation is built automatically by the CI for every pull request. The documentation is hosted on [ReadTheDocs](https://pypsa-earth-status.readthedocs.io/en/latest/).
+
+## No-Code
+
+Instead of contributing code there are alternatives to support the PyPSA-Earth-Status goals.
+You can fund projects, supervise people, provide reference data and support us with outreach activities or events.
+Check out our [website](https://pypsa-meets-earth.github.io) for more details.
```

Comment:
Trying to re-arrange the suggested contributions according to my understanding of their relevance (feel free to disagree!):

```suggestion
You can provide reference data and share your professional insights relevant for
validation, help fellow modellers asking for assistance in [Discord](https://discord.gg/AnuJBk23FU) Support channel,
contribute into outreach activities and events. Feel free to use [this form](https://docs.google.com/forms/d/1udHf6W34YI0UNg3iwQs_-oeKsyj-dzJOtETZ_-RSUhw/edit) to share
your insights and check out our [website](https://pypsa-meets-earth.github.io)
for more details on ways to engage.
```